addons/crm/models/crm_lead.py

```diff
@@ -1405,6 +1408,51 @@ def _merge_dependences_calendar_events(self, opportunities):
             'opportunity_id': self.id,
         })
 
+    def _merge_followers(self, opportunities):
+        """Add the followers into the destination lead if they post a message in the last 7 days.
+
+        :param opportunities : Record<crm.lead> of opportunities to transfer
+        :return: {old_lead_id: Record<mail.followers>} Followers which have been added in
+            the destination lead grouped by source lead ID.
+        """
+        self.ensure_one()
+
+        self.env['mail.message'].flush()
+        self.env['mail.followers'].flush()
+
+        # Get the active followers (followers whose partner post a message on the
+        # leads in the last 7 days) which should be moved on the destination lead
+        self.env.cr.execute(
+            '''
+            SELECT MAX(mf.id) AS id
+              FROM mail_followers AS mf
+              JOIN mail_message AS mm
+                ON mm.author_id = mf.partner_id
+               AND mm.res_id = mf.res_id
+               AND mm.model = 'crm.lead'
+               AND mm.date > NOW() - INTERVAL '7 DAY'
```

Comment:
Thinking of it, if we add message authors only (which is quite restrictive), we could probably increase the interval to two months or something like that. Indeed if we merge a lead staging since a few times, probably is high that there was no message in the last few days. What do you think ?

Reply:
Do not really know in practice, maybe it's safer 2 months, I make the change and will notify LNA